### PR TITLE
should not deploy ops-portal-secret

### PR DESCRIPTION
**Pull request by alejandroEsc**
I may be mistaken, but other tools will create this secret, we probably should not create it in production as it will break and fail to deploy.

### CODE REVIEW

File: staging/traefik-forward-auth/templates/secret.yaml [hunk 1]
```diff
@@ -6,6 +6,8 @@ type: Opaque
 data:
   clientSecret: {{ required "traefikForwardAuth.clientSecret is required" .Values.traefikForwardAuth.clientSecret.value | b64enc | quote }}
   secret: {{ required "traefikForwardAuth.secret is required" .Values.traefikForwardAuth.secret | b64enc | quote }}
+
+{{- if .Values.traefikForwardAuth.opsPortalSecret.enable }}
```
Comment (hectorj2f) [inline staging/traefik-forward-auth/templates/secret.yaml]: @jongiddy @mhrabovcin this sounds like an appropriate change to me. Does it also seem to you ?

Comment (jongiddy) [reply staging/traefik-forward-auth/templates/secret.yaml]: Putting knowledge of the Ops Portal into a chart for a different service seems strange to me. I expect a chart to be generic. Given that this has been done anyway, I don't object to this particular change.

Comment (hectorj2f) [reply staging/traefik-forward-auth/templates/secret.yaml]: okay, I am thinking about it ☕️...  Let's delete this secret here and so the configuration. We don't need it here.

**Comment by alejandroEsc** [comment]
@hectorj2f done. I agree. Also should this chart be in /staging?

**Pull request merged**